spec/models/manageiq/providers/cloud_manager/auth_key_pair_spec.rb

```diff
@@ -9,7 +9,6 @@
 
     # TODO(maufart): do we have any special approach to test module methods separately?
     it 'forces implement methods' do
-      expect { subject.class.create_key_pair ems.id, {} }.to raise_error NotImplementedError
```

Comment:
Since you're going to ask about this one too, @jrafanie, this test is no longer valid, and we no longer have a class that can validly test this so it got 🔥 🔪

Reply:
Why would I ask about deleting code?  🗑 🚽

Reply:
because fixing the broken tests by removing them doesn't usually fly, oddly enough